Upstream-Status: Submitted [https://gitlab.freedesktop.org/xorg/xserver/merge_requests/22]
Signed-off-by: Ross Burton <ross.burton@intel.com>

From 5f65a6246fe752764045dd1e38912f1dccec71e4 Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@intel.com>
Date: Thu, 20 Sep 2018 20:12:24 +0100
Subject: [PATCH] xorg-server.m4: just all cflags instead of just sdkdir

Instead of fetching just the sdkdir variable of xorg-server using pkg-config,
simply get all of the CFLAGS.  Aside from completeness, this helps builds in
sysroots as pkg-config knows what to do with --cflags but doesn't remap
arbitrary variables.

Signed-off-by: Ross Burton <ross.burton@intel.com>
---
 xorg-server.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xorg-server.m4 b/xorg-server.m4
index 18255b91a..195bda5d8 100644
--- a/xorg-server.m4
+++ b/xorg-server.m4
@@ -31,7 +31,7 @@ dnl
 AC_DEFUN([XORG_DRIVER_CHECK_EXT],[
 	AC_REQUIRE([PKG_PROG_PKG_CONFIG])
 	SAVE_CFLAGS="$CFLAGS"
-	CFLAGS="$CFLAGS -I`$PKG_CONFIG --variable=sdkdir xorg-server`"
+	CFLAGS="$CFLAGS `$PKG_CONFIG --cflags xorg-server`"
 	AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
 #include "xorg-server.h"
 #if !defined $1
-- 
2.11.0

